server/controllers/userStudiesController/index.js

```diff
@@ -0,0 +1,29 @@
+import UserModel from '../../models/UserModel'
+import UserStudiesModel from '../../models/UserStudiesModel'
+
+const userStudiesController = {
+  index: async (req, res) => {
+    try {
+      const { appDb } = req.app.locals
+      const parsedSortParams = Object.keys(req.query.sort).length
```

Comment:
This is going to break if `req.query.sort` is undefined

![Screenshot 2024-04-10 at 9 22 17 AM](https://github.com/AMP-SCZ/dpdash/assets/2905145/7c142aeb-445b-4761-9c8e-be7c39b3a097)